src/content/docs/how-to/develop-apps/better-panic.md

```diff
@@ -148,83 +148,9 @@ thread: main
 This is not particularly useful to show to the average user. We'll discuss better solutions for what
 to show the users of your application in the following subsections.
 
-### color-eyre panic hook
+## color-eyre panic hook
```

Comment:
I'd say we can drop this section header and content entirely.

A new reader will not know what we are talking about here and an old reader can see this PR to see why it was moved if they are interested. Also it'll be visible from the sidebar anyway. 

With the new order recommended in another comment, a reader would have encountered the `color_eyre` section first.

Reply:
sounds good